.circleci/config.yml

```diff
@@ -0,0 +1,94 @@
+version: 2
+
+references:
+  container_config: &container_config
+    machine: true
```

Comment:
We need to use machine, because we need to use testcontainers, because some tests need to stop and start the Zookeeper.